docs/modeling/models.rst

```diff
@@ -942,7 +942,7 @@ format. This can be useful in many contexts, one of which is the implementation
 
 Serializing a model to disk is possible by assigning the object to ``AsdfFile.tree``:
 
-.. doctest-requires:: asdf
+.. doctest-requires:: asdf<3.0.0.dev
```

Comment:
I don't think this is supported. Would be nice if it does though. Let's see...

Reply:
modeling passed: https://github.com/astropy/astropy/actions/runs/4929120049/jobs/8808352373?pr=14797#step:10:1646

However coodinates index.rst failed:
```
346 .. doctest-remote-data::
347 
348     >>> EarthLocation.of_address('1002 Holy Grail Court, St. Louis, MO')  # doctest: +FLOAT_CMP
349     <EarthLocation (-26769.86528679, -4997007.71191864, 3950273.57633915) m>
350     >>> EarthLocation.of_address('Danbury, CT')  # doctest: +FLOAT_CMP
Expected:
    <EarthLocation (1362610.66896362, -4590755.48088484, 4198817.69912853) m>
Got:
    <EarthLocation (1364606.6451165, -4593292.9428273, 4195415.93695139) m>

/home/runner/work/astropy/astropy/docs/coordinates/index.rst:350: DocTestFailure
```

Is this a known issue?

Reply:
Testing 5.2.2 I get the 'Got' result:
```python
In [22]: import astropy.coordinates

In [23]: astropy.coordinates.EarthLocation.of_address('Danbury, CT')
Out[23]: <EarthLocation (1364606.6451165, -4593292.9428273, 4195415.93695139) m>

In [24]: astropy.__version__
Out[24]: '5.2.2'
```

Did someone move Danbury?

Reply:
@saimn and @bsipocz , I didn't know we can specify version comparison in `.. doctest-requires::`. When did this happen? 😸

Reply:
Not sure(I'm afk), is it not listed in the changelog?

Reply:
I didn't come through the change log. Maybe I should. The README didn't mention.

Reply:
Would you mind opening a quick PR then? We should be more diligent with requiring the readme updates with the enhancements, but afaik we're not insistent enough

Reply:
Yeah, it is on my todo list. 😸

Reply:
Opened https://github.com/astropy/pytest-doctestplus/pull/198